src/main/java/frc/robot/commands/autos/OneFiveBalanceCenter.java

```diff
@@ -62,12 +48,7 @@ public OneFiveBalanceCenter(
         new ParallelDeadlineGroup(
             new SequentialCommandGroup(
                 new WaitCommand(0.5),
-                new RunCommand(
-                        () -> {
-                          drive.rotateOrKeepHeading(NORM_SPEED_INTAKING, 0, 0, false, -1);
-                        })
-                    .until(
-                        () -> (drive.getPose().getX() - startXMeters) > DISTANCE_AT_CONE_METERS)),
+                new DriveDistance(drive, NORM_SPEED_INTAKING, DISTANCE_AT_CONE_METERS, 0, false)),
```

Comment:
This should now be a relative distance, not a distance from startX, maybe like 1 meter or 1.5 meters?